README.md

```diff
@@ -93,9 +89,9 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration, resources/assets used
-- etc
+- Free code camp
+- Fcc cat photo app
+- Link to fcc cat photo app (https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-1)
```

Comment:
Why do you have 3 different lines here when everything is from the same source??